src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -449,6 +450,35 @@
    (fn [{:keys [refs this]}]
      (.addEventListener (@refs "filter-field") "search" #(when (= (.-value (.-currentTarget %)) "") (react/call :apply-filter this))))})
 
+(react/defc FacetFilter
+  {:render
+   (fn [{:keys [props state]}]
+     [:div (:title props)]
+     [:div ;{:style {:display "flex" :justifyContent "space-between"}}
+      (map
+        (fn [{:keys [key doc_count]}]
+          ([Checkbox
+            {;:ref key
+             :label key
+             }]
+           [:span {:style {:background (:background-light style/colors)  :alignItems "right" }} doc_count]))
+        (:buckets @state))])
+
+   ;:get-checked
+   ;(fn [] )
+   ;:apply-filter
+   ;(fn [{:keys [props refs]}]
+   ;  ((:on-filter props) {this (react/call :get-checked-list this)}))
+   ;:update-from-results
+   ;(fn [{:keys [props state]} aggdata]
+   ;  ;; diff between update and assoc?
+   ;  (swap! state assoc :buckets (:buckets (:results aggdata)) :other_docs (:numOtherDocs (:results aggdata))))
+   ;:component-did-mount
+   ;(fn [{:keys [refs this]}]
+   ;  (.addEventListener (@refs "filter-field") "search" #(when (= (.-value (.-currentTarget %)) "") (react/call :apply-filter this))))
+
+   })
```

Comment:
sorry, none of the changes in component are used. i'm going to revert it back to what is in develop